### ISSUE TITLE
Trouble on install

### ISSUE THREAD

**Issue by sukha** [itota/trac-subtickets-plugin#3]
Hello,

I am very interested in trying out your plugin. I followed the install instructions on your page for a command line install. All seemed to complete without error, but my tickets looks the same as before (the subticket section is not added to the ticket below the description section). I have restarted Trac a couple times as well.

On the Manage Plugins page of the Admin tab Subtickets is not listed as an installed plugin, but when I try to add it using the Intsall Plugin pane on that page I receive a message that the plugin is already installed.

I am using version 0.11.3. Any ideas what might have gone wrong here?

Thanks for your time.

**Comment by itota** [comment]
It seems to work for me with 0.11.3. What does the trac log say?

**Comment by sukha** [comment]
Well, here is the output on the command line during the install:

server:TracSubTicketsPlugin-0.1.0 root# easy_install dist/*.egg
Processing TracSubTicketsPlugin-0.1.0-py2.6.egg
Copying TracSubTicketsPlugin-0.1.0-py2.6.egg to /Library/Python/2.5/site-packages
Adding TracSubTicketsPlugin 0.1.0 to easy-install.pth file
Installing check-trac-subtickets script to /usr/local/bin

Installed /Library/Python/2.5/site-packages/TracSubTicketsPlugin-0.1.0-py2.6.egg
Processing dependencies for TracSubTicketsPlugin==0.1.0
Searching for TracSubTicketsPlugin==0.1.0
Reading http://pypi.python.org/simple/TracSubTicketsPlugin/
Couldn't find index page for 'TracSubTicketsPlugin' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading http://pypi.python.org/simple/
No local packages or download links found for TracSubTicketsPlugin==0.1.0
error: Could not find suitable distribution for Requirement.parse('TracSubTicketsPlugin==0.1.0')

Really nothing to speak of in either trac_errors or trac_server. I didn't set up the system so I don't know if there was any special configuration to the logs. I could look for more command line out put. I did backup Python and the env before the install. I rolled back to the originals and tried the install again. The output I sent is from the second clean attempt to install.

Thanks for your response. I very much appreciate your efforts.
